apps/ledger-live-mobile/ios/ledgerlivemobile.xcodeproj/project.pbxproj

```diff
@@ -817,8 +817,14 @@
 				);
 				MTL_ENABLE_DEBUG_INFO = NO;
 				ONLY_ACTIVE_ARCH = YES;
-				OTHER_CFLAGS = "$(inherited)  ";
-				OTHER_CPLUSPLUSFLAGS = "$(inherited)  ";
+				OTHER_CFLAGS = (
```

Comment:
IMO we should stop committing formatting changes on this file. It could creates unnecessary conflicts and it changes depending on whether we build with the npm script or xcode.

Reply:
Yep true, I was asking myself why this happened !